docs/en/debug/failure_injection.md

```diff
@@ -61,12 +61,19 @@ where:
 - _instance number_ (optional): Instance number of affected sensor.
   0 (default) indicates all sensors of specified type.
 
-### Example
+## MAVSDK Failure Plugin
+
+The [MAVSDK failure plugin](https://mavsdk.mavlink.io/main/en/cpp/api_reference/classmavsdk_1_1_failure.html) can be used to programmatically inject failures.
+It is used in [PX4 Integration Testing](../test_and_ci/integration_testing_mavsdk.md) to simulate failure cases (for example, see [PX4-Autopilot/test/mavsdk_tests/autopilot_tester.cpp](https://github.com/PX4/PX4-Autopilot/blob/main/test/mavsdk_tests/autopilot_tester.cpp)).
+
+The plugin API is a direct mapping of the failure command shown above, with a few additional error signals related to the connection.
+
+## Example: RC signal
```

Comment:
FYI only

I moved these to the end and up a level, even though they are associated with the system command and not the SDK. That allows them to appear in the TOC which is more important IMO. Note that I don't see this as each example explains how it is used.

I also renumbered to use 1, 2, 3. Previously I used 1 for all items and allowed auto-numbering. Because most people use VSCode and prettier to renumber, doesn't seem necessary to have this convention.